### PR TITLE
Fix #272: Individualization clarifications

### LINKED ISSUE #272: Individualization section is unclear
I recently had some confusion about individualization.  I eventually figured things out, but I think it should be made clear that:

* CDM implementations only support one type of individualization or the other
* The CDM implementation dictates the mechanism used (if any), not the user agent, nor the application
* Direct individualization goes to a central _non-origin-specific_ server, but still produces an _origin-specific_ identifier

### PR DESCRIPTION
**Pull request by mwatson2**

### CODE REVIEW

**Comment by joeyparrish** [comment]
LGTM

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -3911,13 +3911,16 @@
...
             The mechanisms, flow, and restrictions for the two types of individualization are different, as described in the following sections.
           </p>
 
           <section id="direct-individualization">
             <h5>Direct Individualization</h5>
-            <p>Direct Individualization is performed between the <a def-id="cdm"></a> and an origin- and application-independent server.
+            <p>
+              Direct Individualization is performed between the <a def-id="cdm"></a> and an origin- and application-independent server.
+              Although the server is origin-independent, the result of the individualization enables the CDM to provide origin-specific identifiers per the other
+              requirements of this specification.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: I noticed a typo at 3936: I believe “origin(s)-” in the last paragraph of this section is _not_ supposed to have a dash. The dash in “orign-” _is_ correct.

**Comment by ddorwin** [comment]
LGTM

File: encrypted-media-respec.html [hunk 2]
```diff
@@ -3911,13 +3911,16 @@
             The resulting identifier(s) MUST be <a def-id="non-associable-by-application"></a> and <a href="#uses-distinctive-identifiers">use of them</a> MUST only be exposed to a <a href="#per-origin-identifiers">single origin</a>.
             This process MAY be performed multiple times, such as after identifier(s) are <a href="#allow-identifiers-cleared">cleared</a>.
           </p>
-          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a>.
+          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a> depending on the CDM implementation.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: We might add something like "and as dictated by the requirements of this specification." IOW, the CDM implementation in combination with the requirements (i.e. the type of data being exposed) determine which is used.

**Pull request merged**